capstone-frontend/src/LeftSideBar.test.js

```diff
@@ -14,7 +14,7 @@ const container = document.createElement("div");
 document.body.appendChild(container);
 
 test('renders sidebar test', () => {
-  render(<Router> <LeftSideBar /> </Router>, container);
+  render(<App />, container);
```

Comment:
What was the rationale for changing this? Is it because `LeftSideBar` needs props from `App`? If so you could manually send the props to LeftSideBar in the testing. Just think it makes more sense to render components in isolation, as if there is an error in any component this will fail to build, which could make debugging difficult.

Reply:
Yeah, we need to render App in order to send the Props down, else the test ``` const createBookListBtn = document.getElementById("create-list-modal");
  expect(createBookListBtn).toBeInTheDocument();
  expect(createBookListBtn.innerHTML).toEqual(" Create New List "); ``` will fail.

Reply:
I will make another test do this test only render the individual component.

Reply:
Ok, I mangage to make it work without rendering the entire ```<App />```